runtests.py

```diff
@@ -2,17 +2,9 @@
 import subprocess
 from subprocess import Popen
 from os import system
-from sys import argv, exit, platform, executable, version_info
+from sys import argv, exit
 
-
-# Use the Python provided to execute the script, or fall back to a sane default
-if version_info >= (3, 5, 0):
-    python_name = executable
-else:
-    if platform == 'win32':
-        python_name = 'py -3'
-    else:
-        python_name = 'python3'
+python_name = 'python'
```

Comment:
I assume modifications of this file is for local testing on your machine. However, this generally should not be committed to the PR.

Reply:
Of course. Sorry I let this slip through.